package.json

```diff
@@ -27,16 +28,20 @@
     "@types/express": "^4.0.37",
     "@types/express-session": "^1.15.3",
     "@types/nunjucks": "^3.1.3",
+    "@types/redis": "^2.8.14",
     "del": "^5.1.0",
-    "gulp": "^4.0.0",
+    "inversify": "^5.0.1",
```

Comment:
I think it should not be in dev dependencies cause DI is used at runtime. You can test it by removing node modules, running `npm install --production` and finally starting the app.